cloudformation/vpc.yml

```diff
@@ -66,6 +66,39 @@ Resources:
           QueryStringsConfig:
             QueryStringBehavior: all
 
+  longLivedCachingHeaderPolicy:
+    Type: AWS::CloudFront::ResponseHeadersPolicy
+    Properties:
+      ResponseHeadersPolicyConfig:
+        Name: CustomLongLivedCachingHeaderPolicy
```

Comment:
You are going to have to give these a new unique name.  Maybe just make it plural:
```
- CustomLongLivedCachingHeaderPolicy
+ CustomLongLivedCachingHeadersPolicy
```
Without a unique name you are going to run into the same issues trying to apply this in production.